presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2153,6 +2153,20 @@ public void revokeTablePrivileges(ConnectorSession session, SchemaTableName sche
         return result.build();
     }
 
+    @Override
+    public void commitPartitionForCreate(ConnectorSession session, ConnectorOutputTableHandle tableHandle, Collection<Slice> fragments)
```

Comment:
We should rename it to `commitPartitionForCreateTable` to be consistent with the `beginCreateTable`  and the `finishCreateTable`